code/datums/jobs/job/job.dm

```diff
@@ -143,8 +143,8 @@ GLOBAL_PROTECT(exp_specialmap)
 /datum/job/proc/radio_help_message(mob/M)
 	to_chat(M, {"
 <span class='role_body'>|______________________|</span>
-<span class='role_header'>You are a: [title]!</span>
-<span class='role_body'>As a [title] you answer to [supervisors]. Special circumstances may change this.</span>
+<span class='role_header'>You are a / an [title]!</span>
+<span class='role_body'>As a / an <b>[title]</b> you answer to [supervisors]. Special circumstances may change this.</span>
```

Comment:
would the byond \an  text macro work here?
What it does is as I understand is automatically add a/an as needed

Reply:
I guess so, but I do not know how to do that.